labelparse.py

```diff
@@ -37,19 +33,19 @@ def repo_fetch():
 
     for repo in dtk_repo_names:
         response = requests.get(
-            'https://api.github.com/repos/newrelic/' + str(repo) + '/issues')
-        if response.status_code == 200:
+            f'https://api.github.com/repos/newrelic/{repo}/issues'
+        )
+        if response:
             for issue in response.json():
                 print_issue(issue)
-                for label in issue['labels']:
-                    if label['name'] in labels_count:
-                        labels_count[label['name']] += 1
-                    else:
-                        labels_count[label['name']] = 1
+                for label in issue.get('labels', []):
+                    labels_count[label['name']] = (
+                        labels_count.setdefault(label['name'], 0) + 1
+                    )
         else:
             print('Error: Not Found.')
 
     print(labels_count)
 
-
-repo_fetch()
+if __name__ == '__main__':
```

Comment:
If you don't have this check, then `repo_fetch()` will be called any time this module is loaded. So, for example, if it's imported into another module (`import labelparse`), it'll run that function. Better is to check to see if this is the main module and only auto-execute that function in that case. Then, if you import it, you can manually run `repo_fetch()` at the correct time, instead of automatically at import-time.

(Fair enough if you never expect it to be imported or used in any other way than `__main__`... but better safe than sorry? 😅)